frontend/javascripts/test/snapshots/public/test-bundle/test/libs/nml.spec.js.md

```diff
@@ -21,7 +21,7 @@ Generated by [AVA](https://ava.li).
         <editPosition x="0" y="0" z="0" />␊
         <editRotation xRot="0" yRot="0" zRot="180" />␊
         <zoomLevel zoom="1.3" />␊
-        <userBoundingBox topLeftX="5" topLeftY="5" topLeftZ="5" width="245" height="245" depth="245" />␊
+        <userBoundingBox topLeftX="5" topLeftY="5" topLeftZ="5" width="245" height="245" depth="245" color.r="255" color.g="0" color.b="0" color.a="1" id="10" name="Test Bounding Box" isVisible="true" />␊
```

Comment:
@MichaelBuessemeyer These colors seem to be in the wrong range, same for the trees actually (for example line 86). Could you check whether the default data in the tests is simply wrong or whether there is another error that's causing this?

Reply:
Done :smile: